addons/arrays/fnc_findMin.sqf

```diff
@@ -24,13 +24,12 @@ Author:
 #include "script_component.hpp"
 SCRIPT(findMin);
 
-private ["_index"];
-
 if (!IS_ARRAY(_this)) exitWith {nil};
 if (_this isEqualTo []) exitWith {nil};
 
 params ["_min"];
-_index = 0;
+
+private _index = 0;
 
 {
     if (isNil "_x" || {(typeName _x) != (typeName 0)}) exitWith {_max = nil; _index = nil;};
```

Comment:
Yep, that's what I referred to with "and in the last isNil check in the file aswell". Good to see the separate PR with the fix. I'll review them ASAP.